terra/assetlist.json

```diff
@@ -1,93 +1,4605 @@
-        "logo_URIs": {
-          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/ust.png"
-        },
-        "coingecko_id": "terrausd"
+      "coingecko_id": "terra-luna"
+    },
+    {
+      "description": "The USD stablecoin of Terra.",
+      "denom_units": [
+        {
+          "denom": "uusd",
+          "exponent": 0,
+          "aliases": [
+            "microusd"
+          ]
+        },
+        {
+          "denom": "musd",
+          "exponent": 3,
+          "aliases": [
+            "milliusd"
+          ]
+        },
+        {
+          "denom": "ust",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "uusd",
+      "name": "TerraUSD",
+      "display": "ust",
+      "symbol": "UST",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/ust.svg",
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/ust.png"
+      },
+      "coingecko_id": "terrausd"
+    },
+    {
+      "description": "The KRW stablecoin of Terra.",
+      "denom_units": [
+        {
+          "denom": "ukrw",
+          "exponent": 0,
+          "aliases": [
+            "microkrw"
+          ]
+        },
+        {
+          "denom": "mkrw",
+          "exponent": 3,
+          "aliases": [
+            "millikrw"
+          ]
+        },
+        {
+          "denom": "krt",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "ukrw",
+      "name": "TerraKRW",
+      "display": "krt",
+      "symbol": "KRT",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/krt.svg",
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/krt.png"
       },
-      {
-        "description": "The KRW stablecoin of Terra.",
-        "denom_units": [
-          {
-            "denom": "ukrw",
-            "exponent": 0
-          },
-          {
-            "denom": "krt",
-            "exponent": 6
-          }
-        ],
-        "base": "ukrw",
-        "name": "TerraKRW",
-        "display": "krt",
-        "symbol": "KRT",
-        "logo_URIs": {
-          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/krt.png"
-        },
-        "coingecko_id": "terrakrw"
+      "coingecko_id": "terrakrw"
+    },
+    {
+      "description": "The governance token of the White Whale protocol on Terra.",
+      "type_asset": "cw20",
+      "address": "terra1php5m8a6qd68z02t3zpw4jv2pj4vgw4wz0t8mz",
+      "denom_units": [
+        {
+          "denom": "cw20:terra1php5m8a6qd68z02t3zpw4jv2pj4vgw4wz0t8mz",
+          "exponent": 0
+        },
+        {
+          "denom": "whale",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:terra1php5m8a6qd68z02t3zpw4jv2pj4vgw4wz0t8mz",
+      "name": "White Token",
```

Comment:
pretty sure this name should be "Whale Token", which is what I see on Terra's list.

Reply:
my bad, made a typo while manually editing this one. fixed!